CONTRIBUTING.rst

```diff
@@ -115,19 +115,33 @@ For a more detailed discussion, read these :doc:`detailed documents
 
    * A pull request must be approved by two core team members before merging.
 
-5. Document changes
+.. _documenting-changes:
 
-   If your change introduces any API modifications, please update
-   ``doc/release/release_dev.rst``.
+5. Document changes
 
    If your change introduces a deprecation, add a reminder to ``TODO.txt``
    for the team to remove the deprecated functionality in the future.
 
+   If your change introduces any API modifications, or impacts users directly
+   (e.g. bug fixes, performance improvements) please describe the change inside
+   the pull request description inside a code block which is compiled by
+   `changelist <https://github.com/scientific-python/changelist>`_ into the
+   release notes::
+
+       ```release-note
+       One or two sentences describing the change in imperative mood.
+       ```
+
+   For trivial changes that are adequately summarized by the PR title, this
+   block can be left empty (``...``). If a public function, class or parameter
+   is referenced the the full import path should be used to do so, e.g.
+   ``skimage.submodule.function``. Refer to :doc:`/release_notes/index` for
+   examples.
```

Comment:
```suggestion
   By default, `changelist <https://github.com/scientific-python/changelist>`_ uses the
   PR title to summarize your change in the release notes.
   
   If you would like to add a longer description of your change in the release notes,
   you may do so inside a dedicated code block::

       ```release-note
       Write a longer description of the change here.
       ```

   When referencing a public function, class, or parameter, please use the full import
   path, e.g., ``skimage.submodule.function``. Refer to :doc:`/release_notes/index` for
   examples.
```

Reply:
Waiting for your thoughts on https://github.com/scikit-image/scikit-image/pull/7267/files#r1429156737 until addressing this.

Reply:
Sure. Sorry for submitting a large *block* of suggestions; my intention was originally to *describe* my ideas in full sentences and bullet points, but then I thought that writing them out might be clear enough.

Reply:
Ok, I've just re-reviewed this PR and I stand by this suggestion; I have updated/resolved my other comments. Thanks!